example/config/deploy.rb

```diff
@@ -4,3 +4,4 @@
 set :application, 'my_app_name'
 set :repo_url, 'git@github.com:MindscapeHQ/capistrano-raygun-deployment.git'
 set :deploy_to, '/var/www/my_app_name'
+set :raygun_deployment_environments, [:production]
```

Comment:
The big issue here is instead requiring raygun_api_key, just detect if it's declared and execute the process. Maybe just put a message like `Raygun is not being executed for this deployment`... or something like that.